src/status/components/StatusIndicator.tsx

```diff
@@ -1,8 +1,22 @@
-import styled, { DefaultTheme, ThemedStyledProps } from 'styled-components';
+import styled, { DefaultTheme, ThemedStyledProps, keyframes } from 'styled-components';
 import { State } from '../ServerStatus';
 
 export type Props = ThemedStyledProps<State, DefaultTheme>;
 
+const fadeInfadeOut =
```

Comment:
const is ok but we can treat it as a reusable mixin - please put it into some common stuff

Reply:
moved to the Theme